articles/root-cause-analysis.md

```diff
@@ -0,0 +1,79 @@
+---
+authorName: Matthias Meid
+authorGithubUsername: matthias-meid
+issue: 98
+title: You Always Have Time for a Proper Root Cause Analysis
+---
+# {{page.title}}
+
+A production issue will pop up, even in solid software. [Don't panic.](https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#Don't_Panic)
+
+We're more than eager to resolve the issue quickly. We should be, yet we should never stop being careful and analytical. Otherwise we might deploy another embarassing bug or suppress the problem without solving it, making everything worse.
```

Comment:
"We're more than eager" -> "We as software engineers are more than eager" (siehe Kommentar bei "Don't have the BA do it for you"